frontend/src/App.tsx

```diff
@@ -49,63 +49,144 @@ import '@ionic/react/css/display.css'
+    case AuthState.Registering:
+      return (
+        <RegisterProvider>
+          <RegisterPage setAuthState={setAuthState} />
+        </RegisterProvider>
+      )
+    case AuthState.ConfirmSignUp:
+      return (
+        <RegisterProvider>
+          <ConfirmSignUpPage />
+        </RegisterProvider>
+      )
+    case AuthState.LoggedIn:
+      return (
+        <IonApp>
+          <IonReactRouter>
+            <IonTabs>
+              <IonRouterOutlet>
+                {/* Moments routes  */}
+                <Route exact path={'/moments'} component={MomentsListView} />
+                <Route
+                  path={`/moments/details/:id`}
+                  component={MomentsDetailView}
+                />
+                <Route
+                  path={`/moments/create`}
+                  component={MomentsCreateNewView}
+                />
+
+                {/* Reflections routes */}
+                <Route
+                  exact
+                  path={'/reflections'}
+                  component={ReflectionsListView}
+                />
+                <Route
+                  path={`/reflections/details/:id`}
+                  component={ReflectionsDetailView}
+                />
+                <Route
+                  path={`/reflections/create`}
+                  component={ReflectionsCreateNewView}
+                />
+
+                {/* profile routes */}
+                <Route exact path={'/profile'} component={ProfileDetailView} />
+                <Route
+                  path={`/profile/changepassword`}
+                  component={ProfileChangePasswordView}
+                />
+
+                {/* progress routes */}
+                <Route
+                  exact
+                  path={'/progress'}
+                  component={ProgressDetailView}
+                />
+
+                {/* redirect from home */}
+                <Route exact path={'/'}>
+                  <Redirect to="/moments" />
+                </Route>
+              </IonRouterOutlet>
+              <IonTabBar slot="bottom">
+                <IonTabButton tab="moments" href="/moments">
+                  <IonIcon icon={image} />
+                  <IonLabel>Momente</IonLabel>
+                </IonTabButton>
+                <IonTabButton tab="reflections" href="/reflections">
+                  <IonIcon icon={albums} />
+                  <IonLabel>Reflexionen</IonLabel>
+                </IonTabButton>
+                <IonTabButton tab="profile" href="/profile">
+                  <IonIcon icon={person} />
+                  <IonLabel>Profil</IonLabel>
+                </IonTabButton>
+                <IonTabButton tab="progress" href="/progress">
+                  <IonIcon icon={barChart} />
+                  <IonLabel>Fortschritt</IonLabel>
+                </IonTabButton>
+              </IonTabBar>
+            </IonTabs>
+          </IonReactRouter>
+        </IonApp>
+      )
+  }
+}
+
+export default App
```

Comment:
```suggestion
export default withAuthenticator(App, false, [
  <SignIn />,
  <MyConfirmSignIn />,
  <VerifyContact />,
  <SignUp />,
  <ConfirmSignUp />,
  <ForgotPassword />,
  <RequireNewPassword />,
])
```

```javascript
const MyConfirmSignIn = (props) => {
  //props.authState === "signIn"
  console.log(props) 
  return <h1>MySignIn</h1>
}
```

https://docs.amplify.aws/lib/auth/customui/q/platform/js#customize-text-labels
https://github.com/aws-amplify/amplify-js/blob/df95ea3724eb6406f64b03f25086cd3e8644cb5f/packages/aws-amplify-react/src/Auth/SignUp.tsx


https://user-images.githubusercontent.com/22683980/118869656-f6a59380-b8e5-11eb-8f08-a345d184fa4c.mov